ispdb/ptd.net

```diff
@@ -1,46 +1,60 @@
-      <port>993</port>
-      <socketType>SSL</socketType>
-      <username>%EMAILLOCALPART%</username>
-      <authentication>password-cleartext</authentication>
-    </incomingServer>
-    <incomingServer type="imap">
-      <hostname>promail.ptd.net</hostname>
-      <port>143</port>
-      <socketType>STARTTLS</socketType>
-      <username>%EMAILLOCALPART%</username>
-      <authentication>password-cleartext</authentication>
-    </incomingServer>
-    <incomingServer type="pop3">
-      <hostname>promail.ptd.net</hostname>
-      <port>995</port>
-      <socketType>SSL</socketType>
-      <username>%EMAILLOCALPART%</username>
-      <authentication>password-cleartext</authentication>
-    </incomingServer>
-    <incomingServer type="pop3">
-      <hostname>promail.ptd.net</hostname>
-      <port>110</port>
-      <socketType>STARTTLS</socketType>
-      <username>%EMAILLOCALPART%</username>
-      <authentication>password-cleartext</authentication>
-    </incomingServer>
-    <outgoingServer type="smtp">
-      <hostname>promail.ptd.net</hostname>
-      <port>587</port>
-      <socketType>STARTTLS</socketType>
-      <username>%EMAILLOCALPART%</username>
-      <authentication>password-cleartext</authentication>
-    </outgoingServer>
-    <documentation url="http://www.ptd.net/help-desk/email-communication/how-setup-your-email">
-      <descr lang="en">How to set up email</descr>
-    </documentation>
-  </emailProvider>
+    <emailProvider id="ptd.net">
+        <domain>ptd.net</domain>
+        <displayName>PenTeleData</displayName>
+        <displayShortName>PenTeleData</displayShortName>
+        <incomingServer type="imap">
+            <hostname>promail.ptd.net</hostname>
+            <port>993</port>
+            <socketType>SSL</socketType>
+            <username>%EMAILLOCALPART%</username>
+            <authentication>password-cleartext</authentication>
+        </incomingServer>
+        <incomingServer type="imap">
+            <hostname>promail.ptd.net</hostname>
+            <port>143</port>
+            <socketType>STARTTLS</socketType>
+            <username>%EMAILLOCALPART%</username>
+            <authentication>password-cleartext</authentication>
+        </incomingServer>
+        <incomingServer type="pop3">
+            <hostname>promail.ptd.net</hostname>
+            <port>995</port>
+            <socketType>SSL</socketType>
+            <username>%EMAILLOCALPART%</username>
+            <authentication>password-cleartext</authentication>
+        </incomingServer>
+        <incomingServer type="pop3">
+            <hostname>promail.ptd.net</hostname>
+            <port>110</port>
+            <socketType>STARTTLS</socketType>
+            <username>%EMAILLOCALPART%</username>
+            <authentication>password-cleartext</authentication>
+        </incomingServer>
+        <outgoingServer type="smtp">
+            <hostname>promail.ptd.net</hostname>
+            <port>465</port>
```

Comment:
The linked documentation doesn't list this port as being supported for SMTP. It did work in my test. Still, I wouldn't include it unless it is documented by the provider.

Reply:
@cketti Our policy is: prefer normal (implicit) TLS over STARTTLS whenever possible. It's faster connected. Our policy also is to list all available and working configs, not just those in the official docs.